german/Stopfen/stopfen.md

```diff
@@ -5,6 +5,8 @@ die Ruder?! Sie stellen fest, dass die Paddel nicht im Boot sind.
 
 Was machen Sie nun?
 
+[Im Boot nach was suchen was als Paddel gebraucht werden kann] (.../suchen/suchen.md)
```

Comment:
It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.